### DISCUSSION TITLE
Contribute.md

### DISCUSSION THREAD

**Discussion by Adesoji1** [idom-team/idom#1035] [Ideas]
we need contribute.md on the root directory in github

**Comment by rmorshea** [comment]
We have some docs on this [here](https://reactpy.dev/docs/about/contributor-guide.html) that we [link to in our readme](https://github.com/reactive-python/reactpy#resources). I think we'll probably leave as-is for now, but we're looking at [rewriting the docs with mkdcos](https://github.com/reactive-python/reactpy/issues/863) where I think having a root `CONTRIBUTE.md` file would be easier to do.
